core/templates/dev/head/editor/ExplorationEditor.js

```diff
@@ -172,6 +172,11 @@ oppia.controller('ExplorationEditor', [
     var _ID_TUTORIAL_PREVIEW_TAB = '#tutorialPreviewTab';
     var _ID_TUTORIAL_SAVE_BUTTON = '#tutorialSaveButton';
 
+    var screenWidth = windowDimensionsService.getWidth();
+    if (screenWidth <= 1280) {
+      _ID_TUTORIAL_PREVIEW_TAB = '#tutorialPreviewSmall';
```

Comment:
No, It doesn't work if the window is resized, as it doesn't keep track of the size of screen, you have to refresh for that.I didn't think we needed that.(Why would someone resize in between of editing)